### PR TITLE
Update awards playbook

### PR DESCRIPTION
**Pull request by reylejano**
This PR updates the awards playbook with additional suggestions on how the community can be involved with awards and how SIGs can model awards to their own needs/goals.

### CODE REVIEW

**Comment by reylejano** [comment]
@tokt  @jrsapi @parispittman ptal

File: events/awards/playbook.md [hunk 1]
```diff
@@ -7,9 +7,14 @@ The CNCF typically has awards as part of KubeCon + CloudNativeCon, the Kubernete
...
 ## Ideals and Values
 
-Typically each SIG will nominate two(2) recipients for an award. 
+Typically each SIG will nominate two (2) recipients for an award. 
 Ultimately the individual SIGs are responsible for their award nominations, in certain cases there may be an example of excellent work that requires more than two awards, co-chairs should use their judgement to determine if this is the case. 
 
+SIGs can model their award(s) to the types of contributions a SIG encourages. Some examples are:
+  - Bug squasher
+  - Flake fixer
+  - Most reviews or best reviewer
```
Comment (jberkus) [inline events/awards/playbook.md:16]:
```suggestion
  - Most reviews or best reviewer
  - Team builder
  - KEP wrangler
```
... I'd like to have at least one non-code example there

**Review by jberkus — CHANGES_REQUESTED** [review]
Nice!

Please see suggestion for some non-code examples.

**Pull request merged**